Makefile

```diff
@@ -71,3 +71,7 @@ fmt: fmt-nix fmt-rust fmt-js
 
 .PHONY: fmt-check
 fmt-check: fmt-nix-check fmt-rust-check fmt-js-check
+
+.PHONY: lint
+lint:
+	@cargo clippy -- -D warnings -A clippy::let_underscore_future -A clippy::module_inception -A clippy::op_ref -A clippy::manual_strip -A clippy::missing_safety_doc -A clippy::slow_vector_initialization -A clippy::empty_loop
```

Comment:
I don't think is the preferred way of configuring clippy

https://doc.rust-lang.org/stable/clippy/configuration.html

Reply:
as discussed, clippy config file doesn't support all the linting rules sadly :(